jirate/template_vars.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/python3
+#
+# Jinja2 variable substitution uses glyphs that are native to JIRA so
+# use direct/simple substitution instead.  CPaaS uses @@var@@, so we
+# will follow suit.
```

Comment:
Looking at Jinja docs, it seems those glyphs are just defaults and can be customized by passing parameters to the Environment initialization: https://jinja.palletsprojects.com/en/3.1.x/api/#jinja2.Environment

So we can set our own `block_start_string`, `block_end_string`, `variable_start_string`... and so on to avoid conflicts with [Jira's markup language](https://jira.atlassian.com/secure/WikiRendererHelpAction.jspa?section=all). Which means we can leverage all of Jinja.

I think this would be less comfortable than using 100% default Jinja but a lot more manageable than creating and dealing with our own fully custom deal forever. What do you think?

Reply:
Reasonable. So then, we can use something like:
- {@ - variable start string
- @} - variable end string